lib/src/model.dart

```diff
@@ -507,7 +507,7 @@ class Library extends ModelElement {
 
   String get path => _library.definingCompilationUnit.name;
 
-  String get nameForFile => name.replaceAll(':', '-');
```

Comment:
I'm not a huge fan of this change. If I see a variable called "fileName", I'm going to assume it's the filename. As it stands, it's really just the part of the filename before the suffix.

I used `nameForFile` to make that clearer. There's probably a better name, but I think `fileName` promises too much.